v1.1/create-security-certificates-openssl.md

```diff
@@ -0,0 +1,326 @@
+---
+title: Create Security Certificates
+summary: A secure CockroachDB cluster uses TLS for encrypted inter-node and client-node communication.
+toc: false
+---
+
+A secure CockroachDB cluster uses TLS for encrypted inter-node and client-node communication, which requires CA, node, and client certificates and keys. To create these certificates and keys, use the `cockroach cert` [commands](cockroach-commands.html) with the appropriate subcommands and flags, or use [`openssl` commands](https://wiki.openssl.org/index.php/).
+
+<div class="filters filters-big clearfix">
```

Comment:
Looks like one of my comments might have gotten lost. In addition to renaming these, can you move them to the top of the page, like we do here: https://www.cockroachlabs.com/docs/stable/build-a-python-app-with-cockroachdb.html

Reply:
My reply got lost too :( I replied saying that the first paragraph is common to both the docs, so I would prefer to keep them it at the top, then segue into the links for each doc. But that's just my preference, I am open to changing it too.

Reply:
If you're willing, I'd prefer them at the top, like we've done on most other pages, for example:
- https://www.cockroachlabs.com/docs/stable/build-a-python-app-with-cockroachdb.html
- https://www.cockroachlabs.com/docs/stable/start-a-local-cluster.html
- https://www.cockroachlabs.com/docs/stable/deploy-cockroachdb-on-aws.html